CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Contributing Guidelines
+
+We love improvements to our documentation! Our process for accepting changes has a few steps:
+
+1. If you haven't submitted anything before, and you aren't (yet!) a member of our organization, **fork and clone** the repo:
+
+        $ git clone git@github.com:<your-username>/<repository-name>.git
+
+  Organization members should clone the upsteam repo, instead of working from a personal fork:
+
+      $ git clone git@github.com:datarefuge/<repository-name>.git
```

Comment:
Actually, I am not sure that Github Desktop makes that specific workflow much easier.